build-tokens.js

```diff
@@ -0,0 +1,78 @@
+const StyleDictionary = require('style-dictionary');
```

Comment:
This is a bit of a nitpick, but it feels odd to me to have this just hanging out in the root. Maybe it's because I remember our Theo config starting out that way and then eventually needing its own directory.

Could we potentially follow [Style Dictionary's example](https://github.com/amzn/style-dictionary/tree/main/examples/advanced/custom-transforms) and create a shared directory that includes a `config.json` file for the simple/default options and a `build.js` script for the rest?

Reply:
Good idea!